es-bs-base/scss/mixins/_buttons.scss

```diff
@@ -3,6 +3,141 @@
 // Easily pump out default styles, as well as :hover, :focus, :active,
 // and disabled options for all buttons
 
+@mixin button-v2-variant($variant) {
+  $background: map-get($button-colors, $variant);
+  $border: $background;
+
+  $hover-background: map-get($button-hover-colors, $variant);
+  $hover-border: map-get($button-hover-colors, $variant);
+
+  $active-background: map-get($button-active-colors, $variant);
+  $active-border: map-get($button-active-colors, $variant);
+
+  $focus-background: map-get($button-hover-colors, $variant);
+  $focus-border: map-get($button-focus-border-colors, $variant);
+
+  color: color-yiq($background);
```

Comment:
this is a cool function 👀

Reply:
Yeah it's [being renamed](https://getbootstrap.com/docs/5.0/migration/#sass) to `color-contrast()` in BS5. I basically just copied stuff from the `button-variant` mixin, including this. It's a bit sketchy though - it _happens_ to output the correct color that matches the design in this case, but it may not always in all cases, so we may not want to use it a whole lot, especially in the case of buttons where we're now being very specific about which colors are used in which states. But here, it definitely works, for now.